python-sdk/nuscenes/map_expansion/map_api.py

```diff
@@ -264,10 +273,12 @@ def render_map_patch(self,
         :param figsize: Size of the whole figure.
         :param render_egoposes_range: Whether to render a rectangle around all ego poses.
         :param render_legend: Whether to render the legend of map layers.
+        :param bitmap: Optional BitMap object to render below the other map layers.
```

Comment:
For what it is worth, it is redundant to include "Optional" in the doc when it is already specified in the type hint.

Reply:
Yeah, both the use of `=` (default argument) and `Optional` (type hint) make this redundant, but I think it doesn't hurt to highlight that this addition is optional.